debian/tests/integrationtests

```diff
@@ -14,6 +14,7 @@ if [ -z "$SNAPCRAFT_SLOW_TESTS" ]; then
     SNAPCRAFT_SLOW_TESTS=1
 fi
 
+mkdir -p /home/ubuntu/autopkgtest_tmp
```

Comment:
You use `$HOME` below, can you do so here?

Reply:
Also, who owns this dir now? root?

Reply:
Good point. I've added 777 to the dir.